tests/layer_tests/common/utils/common_utils.py

```diff
@@ -57,6 +57,12 @@ def generate_ir_python_api(coverage=False, **kwargs):
         serialize(ov_model, out_dir)
     else:
         from openvino import convert_model, save_model
+        try:
+            # noinspection PyUnresolvedReferences
+            import openvino_tokenizers  # do not delete, needed for validation of OpenVINO tokenizers extensions
+        except:
+            # CI Jenkins job and ARM64 has no openvino_tokenizers available
```

Comment:
maybe we can throw an exception here if we are on Linux / Windows x86_64 platform?

Reply:
Fixed. Also, we have to add Linux because module is not importable in CI Jenkins job. For that, I created a ticket to address.